packages/map-template/src/atoms/showRoadNameLabels.js

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+const showRoadNameLabelsState = atom({
+    key: 'showRoadNameLabels',
+    default: undefined
+});
+
+export default showRoadNameLabelsState;
```

Comment:
The file name should follow the pattern and be `showRoadNameLabelsState.js` instead 😄

Reply:
✅